### ISSUE TITLE
Sharing Training Loss 

### ISSUE THREAD

**Issue by LibertFan** [openvla/openvla#187]
Could you share the training loss for better reproduce OpenVLA?

Thanks a lot!

**Comment by siddk** [comment]
Yes! Note that all y-axes are on a log scale (with an exception of the action token accuracy graph). The blue line indicates the period of training prior to shifting the data mixture distribution, while purple reflects after. Please let use know if you have any other questions!

<img width="2095" alt="Image" src="https://github.com/user-attachments/assets/c4a3c507-f5b5-43ae-ab9b-7c38f1ac1a9c" />
<img width="2116" alt="Image" src="https://github.com/user-attachments/assets/7b573d7a-1c68-47b7-9833-d75fdf489705" />
<img width="2106" alt="Image" src="https://github.com/user-attachments/assets/e9bed7b8-523f-4fd2-879b-1688ca21d71c" />

**Comment by LukeLIN-web** [comment]
Why here is a big loss drop in 180k steps?

**Comment by siddk** [comment]
Right - so like we mentioned in Section 3.3 of the paper, we changed the mixture of datasets we trained on for the last third of training (see screenshot below). That's the switch from blue to purple at 180K steps!

<img width="1047" alt="Image" src="https://github.com/user-attachments/assets/6f29bc40-ed9c-4b24-b95f-e3d5295cd6f3" />
